rest/src/main/java/org/jboss/pnc/rest/provider/ArtifactProvider.java

```diff
@@ -169,6 +173,13 @@ private String getPublicUrl(Artifact artifact) {
         return queryForCollection(pageIndex, pageSize, sortingRsql, query, withDependantBuildRecordId(buildRecordId));
     }
 
+    public CollectionInfo<ArtifactRest> getAll(int pageIndex, int pageSize, String sortingRsql, String query,
+            Optional<String> sha256, Optional<String> md5, Optional<String> sha1) {
+        logger.debug("HELLO + sha1: " + sha1 + " sha256:" + sha256 + " md5: " + md5);
```

Comment:
Use better debug message

Reply:
forgot to cherry-pick last commit, my bad